src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -283,7 +284,7 @@ private void showCommentLog() {
         uiContext.getMapData().getMapDimensions().height);
     final Image small = uiContext.getMapImage().getSmallMapImage();
     smallView = new MapPanelSmallView(small, model);
-    mapPanel = new MapPanel(data, smallView, uiContext, model);
+    mapPanel = new MapPanel(data, smallView, uiContext, model, this);
```

Comment:
I told you, that pressing ctlr still isn't working properly...
If we replaced the parameter of computescrollspeed with a boolean (which it uses), we could use a Predicate<Boolean> object this::computeScrollSpeed